yarn.lock

```diff
@@ -7974,7 +7981,14 @@ dot-case@^3.0.4:
     no-case "^3.0.4"
     tslib "^2.0.3"
 
-dot-prop@^4.2.0, dot-prop@^5.1.0, dot-prop@^5.2.0:
+dot-prop@^4.2.0:
+  version "4.2.1"
```

Comment:
do you know why we have dot-prop twice in diff version ?

Reply:
Fixing `minimist` version addressed this 🍀